subsystems/ros2/Makefile

```diff
@@ -0,0 +1,27 @@
+SPECFILE_SUBPACKAGE_ROS2_ROLLING=rpm/ros2/rolling/ros2_rolling.spec
+
+.PHONY: dist
+dist: ##             - Creates the QM ros2 package
```

Comment:
the same here

Reply:
@dougsland the do you refer the dependency removal? 
the empty dist? dependency?